pulumi/src/statefulService.ts

```diff
@@ -379,13 +359,13 @@ export async function deployStatefulService(
         },
         spec: {
           entryPoints: ['web', 'websecure'],
-          routes: Object.entries(services)
-            .map(([service, { ports }]) =>
-              ports
+          routes: services
+            .map((serviceEntry) =>
```

Comment:
`serviceEntry` -> `service`

Reply:
Did that on purpose to avoid variable shadowing. `service` is already defined in this scope

Reply:
that's fine, not super concerned about he shadowing (I feel like there is a lot of that in js specifically with these kind of callback functions. could use `svc` or something, but your call. just feel like stylistically it is usual plural.map((singular/shortened))